### PR TITLE
removed homebrew, simplified with packages for git and node

### PR DESCRIPTION
**Pull request by aantonop**
Cleaned up the Mac OS X installation document. 

Removed the homebrew instructions, the idea being that if you already have or know about homebrew, you'd probably just use it to install node, npm and git anyway, and don't need instructions. 

Learned how to squash and how to do a pull request from another branch to make it easier to push changes later, without merge commit.

### CODE REVIEW

File: doc/INSTALL_OSX.md [hunk 1]
```diff
@@ -2,26 +2,23 @@ Installing on Mac OSX
...
 
-The easiest way to install and run on Mac OSX is using Homebrew (http://mxcl.github.io/homebrew/)
+Download the Universal installer for Mac OS X:
 
-Install Homebrew
-----------------
+http://nodejs.org/download/
 
-To install homebrew, if you have not already, try this command:
+_Note: buttercoin requires node >= 10.4_
```
Comment (PartTimeLegend) [inline doc/INSTALL_OSX.md]: `0.10.4` is latest version. Unit tests have run without issue as far back as `0.8` as you can see from [`.travis.yml`](https://github.com/buttercoin/buttercoin/blob/master/.travis.yml)

Comment (enki) [inline doc/INSTALL_OSX.md]: 0.10.4 is needed. i use readstream features that are new in 0.10

Comment (aantonop) [inline doc/INSTALL_OSX.md]: ok will revert that last change. So 0.10.4 minimum.

On Sun, Apr 14, 2013 at 10:50 PM, Paul Bohm notifications@github.comwrote:

> In doc/INSTALL_OSX.md:
> 
> > -To install homebrew, if you have not already, try this command:
> > +_Note: buttercoin requires node >= 10.4_
> 
> 0.10.4 is needed. i use readstream features that are new in 0.10
> … <#13e0c3ead257d1c0_>
> On Sun, Apr 14, 2013 at 7:28 PM, Antony Bailey notifications@github.comwrote:
> In doc/INSTALL_OSX.md: > > -To install homebrew, if you have not already,
> try this command: > +_Note: buttercoin requires node >= 10.4_ 0.10.4 is
> latest version. Unit tests have run without issue as far back as 0.8 as you
> can see from .travis.yml<
> https://github.com/buttercoin/buttercoin/blob/master/.travis.yml> — Reply
> to this email directly or view it on GitHub<
> https://github.com/buttercoin/buttercoin/pull/30/files#r3787751> .
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/buttercoin/buttercoin/pull/30/files#r3788618
> .

## 

Andreas M. Antonopoulos
andreas@antonopoulos.com
Tel: +1 347 744 9404

**Comment by mks-m** [comment]
i still find homebrew install very simple and convenient. it should be at least mentioned that it's possible to install using brew.

**Comment by PartTimeLegend** [comment]
Anyone tested this?

**Comment by aantonop** [comment]
I have twice, but not anyone else AFAIK

On Mon, Apr 15, 2013 at 2:53 PM, Antony Bailey notifications@github.comwrote:

> Anyone tested this?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/buttercoin/buttercoin/pull/30#issuecomment-16413880
> .

## 

Andreas M. Antonopoulos
andreas@antonopoulos.com
Tel: +1 347 744 9404

**Comment by nfx8** [comment]
I've run through the install again. I have installed the packages locally. Then added node_modules to path. Then npm test runs successfully. npm start gives errors and a not implemented message.

0 info it worked if it ends with ok
1 verbose cli [ 'node', '/usr/local/bin/npm', 'start' ]
2 info using npm@1.2.18
3 info using node@v0.10.4
4 verbose read json /Users/ME/repos/buttercoin/package.json
5 verbose run-script [ 'prestart', 'start', 'poststart' ]
6 info prestart buttercoin@0.0.0
7 info start buttercoin@0.0.0
8 verbose unsafe-perm in lifecycle true
9 silly exec sh "-c" "sh start.sh"
10 silly sh,-c,sh start.sh,/Users/ME/repos/buttercoin spawning
11 info buttercoin@0.0.0 Failed to exec start script
12 error buttercoin@0.0.0 start: `sh start.sh`
12 error `sh "-c" "sh start.sh"` failed with 1
13 error Failed at the buttercoin@0.0.0 start script.
13 error This is most likely a problem with the buttercoin package,
13 error not with npm itself.
13 error Tell the author that this fails on your system:
13 error     sh start.sh
13 error You can get their info via:
13 error     npm owner ls buttercoin
13 error There is likely additional logging output above.
14 error System Darwin 12.0.0
15 error command "node" "/usr/local/bin/npm" "start"
16 error cwd /Users/ME/repos/buttercoin
17 error node -v v0.10.4
18 error npm -v 1.2.18
19 error code ELIFECYCLE
20 verbose exit [ 1, true ]

**Comment by PartTimeLegend** [comment]
@aantonop can you look at @nfx8 's issue please?

**Comment by aantonop** [comment]
Asap. Out at the mo

**Comment by nfx8** [comment]
My mistake. Didn't recognize the changes. Works for me.

**Comment by PartTimeLegend** [comment]
ACK

**Pull request merged**